docs/installation.md

```diff
@@ -50,13 +57,16 @@ See [ArchWiki](https://wiki.archlinux.org/index.php/Haskell) for the details of
 
 HLS is available for installation from official binary packages. Use
 
-```
+```bash
 pkg install hs-haskell-language-server
 ```
 
 to install it. At the moment, HLS installed this way only supports the same GHC
 version as the ports one.
 
+Since 1.4.0 version you also can install HLS via [ghcup](#ghcup).
```

Comment:
Nit, but I don't see much point in giving these kind of historical details here. Users probably care about the current state. If we want to provide docs for old versions we can do that with RTD's versioned documentation!

Also, does this add anything over the ghcup section? Nothing in that section says you *can't* use it on FreeBSD, and none of the other linux sections need to say "btw ghcup works", so... maybe it's fine? (or maybe ghcup *doesn't* work on FreeBSD?) Maybe we should just link to something in the ghcup docs about OS support if there is something like that?

Reply:
hmm yeah, it sounds more like an announcement than a pemanent install instruction. But i wanted to stress you can use ghcup although it is not a linux flavour but a unix one. But same could be applicable for windows cause you can use ghcup there nowadays.

Reply:
> Maybe we should just link to something in the ghcup docs about OS support if there is something like that?

That could be a good one

//cc @hasufell

Reply:
```suggestion
```